src/plugins/commander/commands/ucp.js

```diff
@@ -1,21 +1,62 @@
 const Command = require('../structs/Command.js');
+const FormatterPlugin = require('../../fandomizer');
+const XRay = require('x-ray');
 
 class UCPCommand extends Command {
-    constructor(bot) {
-        super(bot);
-        this.aliases = ['ucp'];
-
-        this.shortdesc = `Posts a links to UCP info.`;
-        this.desc = `Posts links to info about Fandom's UCP platform.`;
+    static get deps() {
+        return [
+            FormatterPlugin
+        ];
     }
+    
+	constructor(bot) {
+        super(bot);
+	this.aliases = ['ucp'];
 
-    async call(message) {
-        message.channel.send(`
+	this.shortdesc = `Posts links to UCP info.`;
+	this.desc = `Posts links to information about Fandom's UCP platform. You can optionally get info on a specified script/stylesheet's compatibility status by providing it as an argument.`;
+	this.usages = [
+            '!ucp [script/stylesheet]'
+        ];
+        this.examples = [
+            '!ucp',
+            '!ucp DiscordIntegrator'
+        ];
+        
+        this.linksString = `
 - Help - <https://c.fandom.com/Help:UCP>
 - Information - <https://fandom.zendesk.com/hc/articles/360044776693>
-- Bugs, features, changes - <https://c.fandom.com/User:Noreplyz/UCP>
-        `);
-    }
+- Bugs, features, changes - <https://c.fandom.com/User:Noreplyz/UCP>`;
```

Comment:
Not completely related to this PR, but I think we could use a link to DEV:UCP here.